client/my-sites/checkout/composite-checkout/add-items.ts

```diff
@@ -26,11 +26,11 @@ export function createItemToAddToCart( {
 	isJetpackNotAtomic,
 	isPrivate,
 }: {
-	planSlug: string | undefined;
-	productAlias: string | undefined;
+	planSlug?: string | undefined;
```

Comment:
Can remove ` | undefined`

Reply:
Would that work if I call this with `planSlug: undefined`? For example,

```js
const planSlug = getPlanSlugOrUndefined();
createItemToAddToCart({ planSlug });
```